cleanlab/multiannotator.py

```diff
@@ -427,6 +866,47 @@ def get_majority_vote_label(
     return majority_vote_label.astype("int64")
 
 
+def get_majority_vote_label_ensemble(
+    labels_multiannotator: Union[pd.DataFrame, np.ndarray],
+    pred_probs: np.ndarray,
+    verbose: bool = True,
+) -> np.ndarray:
+    """Returns the majority vote label for each example, aggregated from the labels given by multiple annotators.
+    This function allows for more than one model's predicted probabilites to be pass in to be used as a tie-breaking stategy.
+
+    Parameters
+    ----------
+    labels_multiannotator : pd.DataFrame or np.ndarray
+        2D pandas DataFrame or array of multiple given labels for each example with shape ``(N, M)``,
+        where N is the number of examples and M is the number of annotators.
+        For more details, labels in the same format expected by the :py:func:`get_label_quality_multiannotator <cleanlab.multiannotator.get_label_quality_multiannotator>`.
+    pred_probs : np.ndarray
+        An array of shape ``(P, N, K)`` where P is the number of models, consisting of predicted class probabilities from the ensemble models.
+        Each set of model-predicted probabilities, ``P(label=k|x)`` has shape ``(N, K)``, and is in the same format expected by :py:func:`get_label_quality_scores <cleanlab.rank.get_label_quality_scores>`.
+
+    verbose : bool, optional
+        Important warnings and other printed statements may be suppressed if ``verbose`` is set to ``False``.
+    Returns
+    -------
+    consensus_label: np.ndarray
+        An array of shape ``(N,)`` with the majority vote label aggregated from all annotators.
+
+        In the event of majority vote ties, ties are broken in the following order:
+        using the average model ``pred_probs`` and selecting the class with highest predicted probability,
+        using the empirical class frequencies and selecting the class with highest frequency,
+        using an initial annotator quality score and selecting the class that has been labeled by annotators with higher quality,
+        and lastly by random selection.
+    """
+
+    if pred_probs.ndim != 3:
+        raise ValueError(
+            "pred_probs must be a 3d array containing the predicted probabilites of the ensemble models"
+        )
+
+    avg_pred_probs = np.mean(pred_probs, axis=0)
+    return get_majority_vote_label(labels_multiannotator, avg_pred_probs, verbose)
```

Comment:
I don't think we need to provide this function? 

User can just do: 
```
avg_pred_probs = np.mean(pred_probs, axis=0)
get_majority_vote_label(labels_multiannotator, avg_pred_probs)
```

so IMO this function is as likely to confuse as it is to help (including extra functions always increases potential confusion for users of a package)

Reply:
Removed the extra function!